flink/sbt/sbt.rst

```diff
@@ -0,0 +1,57 @@
+===============================
+Using SBT to create a Flink job
+===============================
+
+Motivation
+----------
+
+Maven is a widely accepted build tool for projects that target the Java platform. The Radicalbit Fast Data Distribution provides an archetype to kickstart your development projects using Maven, but Scala developers may find more compelling to use the Maven alternative of choice for the Scala ecosystem, SBT (Simple Build Tool).
+
+Basic 'archetype'
+-----------------
+
+Currently, SBT does not provide built-in capabilities to scaffold a project like Maven archetypes do.
+As such, some manual work is required to initialize the build definition properly.
+The following steps describe how to create the initial scaffolding for the project.
+All the steps involve the usage of a terminal emulator and require a basic understanding of how to use one.
+
+First of all, let's create a directory for our Flink project::
+
+    mkdir my-flink-project
+
+Then, move into the directory and create a project directory::
+
+    cd my-flink-project
+    mkdir project
+
+To package our application, we'll use the ``sbt-assembly`` plugin. To add it to our build, create and open in your favorite text editor the ``project/assembly.sbt`` file and fill it in with this single line:
+
+.. literalinclude:: /rbp-docs-code/sbt/project/assembly.sbt
```

Comment:
something is wrong with modules, cannot find referenced code for this link and following.